.github/workflows/ubuntu.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.12', '3.11', '3.8', '3.9', '3.10']
+        python-version: ['3.13', '3.12', '3.11', '3.8', '3.9', '3.8']
```

Comment:
Should keep 3.8? Maybe 3.8,  3.10, 3.12 and 3.13 would be enough?

Reply:
I am for dropping 3.8.

Reply:
OK, so ['3.13', '3.12', '3.11', '3.10', '3.9'] in all the workflows?